### DISCUSSION TITLE
Gitflow x Scaled Trunk Base Development: como escolher a melhor abordagem para o seu time

### DISCUSSION THREAD

**Discussion by mlarasusan** [gittogethers/octogatosconf22#106] [Talks in Portuguese]
**Palestrante:** Nivea Martins & Matheus Bitencourt

**Assunto:**

TBD é um modelo de gerenciamento de controle de versão que tem como intuito o aumento da produtividade do time e entrega constante para o usuário final. Nessa apresentação, iremos te contar a jornada para utilizar TDB, as diferenças entre ele e o Gitflow e os motivos pelo qual escolhemos ele como ideal para o nosso time. Afinal, qual é a melhor abordagem? Vamos discutir os prós e contras de cada nessa busca constante por mais produtividade, segurança e integração.

**Comment by pedrolacerda** [comment]
https://nvie.com/posts/a-successful-git-branching-model/

Pra colocar 🔥 na discussão

**Comment by niveamartins** [comment]
Fala, galera! Vim trazer alguns links como a palestra, minhas redes sociais e algumas referências boas.

### Slide

https://www.canva.com/design/DAFMDH3LLFw/vA0w1g5kBwqvR6xl2HADaw/view

### Redes sociais

Linkedin: https://www.linkedin.com/in/nivea-martins/
Instagram Tech: http://instagram.com.br/nivea.tech
Instagram Pessoal: http://instagram.com.br/niveamgs 
Twitter: https://twitter.com/hellonivea

### Referências

1. https://medium.com/@vafrcor2009/gitflow-vs-trunk-based-development-3beff578030b
2. https://www.devbridge.com/articles/branching-strategies-git-flow-vs-trunk-based-development/
3. https://trunkbaseddevelopment.com/
4. https://itrevolution.com/the-three-ways-principles-underpinning-devops/
5. https://ppbruna.medium.com/mais-uma-discuss%C3%A3o-sobre-trunk-based-vs-branch-based-development-b79e57c6419a
